recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_2.8-l4t-r36.3.0.bb

```diff
@@ -53,8 +53,10 @@ def generate_build_timestamp(d):
 BUILD_STRING ?= "${@generate_build_string(d)}"
 BUILDTIMESTAMP ?= "${@generate_build_timestamp(d)}"
 
+DEBUG ?= "0"
```

Comment:
I prefer to have those variables specific to `ATF`.
It should be something as below:
```
ATF_DEBUG ?= "0"
ATF_LOG_LEVEL ?= "20"
```